e2e/src/fullstack.js

```diff
@@ -125,16 +125,17 @@ describe('A full stack', function () {
         it('can be registered as first gatekeeper', async function () {
             // Register worker1 as Gatekeeper
             const info = await pruntime[0].getInfo();
-            await assert.txAccepted(
-                api.tx.sudo.sudo(
-                    api.tx.phalaRegistry.forceRegisterWorker(
-                        hex(info.system.publicKey),
-                        hex(info.system.ecdhPublicKey),
-                        null,
-                    )
-                ),
-                alice,
-            );
+            // await assert.txAccepted(
```

Comment:
Should it be uncommented?

Reply:
Acutally this should remove, I introduced `none` attestation provider, and dev chain will enable it, so no need to force register worker anymore, just do the normal register worker with none attestation report

Reply:
get it. let's remove it.

Reply:
Removed them